pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -36,8 +38,6 @@ import (
 	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
-	"k8s.io/utils/pointer"
-
```

Comment:
You should keep the newline here, these are two different groups.

Reply:
This is the deleted line. The line added on the right is separated by a newline above and below. Should I change the position?

Reply:
I think we can enforce these things via `goimports`:
https://github.com/kubevirt/containerized-data-importer/pull/3097

Reply:
I think we use it already through bazel. It does not work well enough, especially between different 3rd party domains. And there is a need to configure the "self" domain as well.